html final/index.html

```diff
@@ -0,0 +1,201 @@
+<!Doctype html>
+<html ln="en">
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="styles.css">
+        <title> LIBRARY HOME - AUA's AGBU PAPAZIAN LIBRARY </title>
+        <link rel="icon" type="image/png" href="img/aua.png"/>
+    </head>
+    <body>
+        
+        <header>
+            <div class = "top-green">
+                <a href="" class = "home">AUA Home</a>
+            </div>
+            <div class = "sticky">
+                    <a class ="logo" href="">
+                        <img src="img/library-1.png" alt="logo">
+                    </a> <hr class = "big-hr">
```

Comment:
It would be better to have one HTML element on each line, for better readability. 
```suggestion
                    </a>
                    <hr class = "big-hr">
```